### DISCUSSION TITLE
Moving projects from one file to another file in toonsquid

### DISCUSSION THREAD

**Discussion by mcallbee** [keiwando/toonsquid#441] [General]
How do I move a project out of one file and into another within toonsquid?  I can't seem to drag them out and there is no "move" or "copy" function??? Help

**Comment by DinoAwakening** [comment]
https://github.com/keiwando/toonsquid/assets/120270913/5534d04d-b280-4e9a-b525-97484300dcd9

You might had figured it out already, but here how you do it ;D

**Reply by mcallbee** [reply]
Hi Dino,
No, I had not figured it out, but thank you for the help. Just what I needed.

**Reply by mcallbee** [reply]
Perhaps you could help me with another file question....I have a file with 61 projects in it...when I highlight the file for export, the lower options light up, but when I click "export" nothing happens. How do I export an entire file? Why is toon preventing it??

**Reply by DinoAwakening** [reply]
So what happens is that ToonSquid, because of apparently being selected just one file, not identifying it as a folder, highlights all the options available for a single project. So, the thing is, while you can export any project just as normal, multiple projects exported at once is not possible, since they don’t share configuration most of the time, and ToonSquid doesn’t have an engine to fuse them. In short, just like mixing oil and water xd
